Ruby/person.rb

```diff
@@ -1,6 +1,7 @@
 require_relative './decorate'
 
 class Person < Nameable
+  @rentals = []
   def initialize(age, name = 'Unknown', parent_permision = true)
     super()
```

Comment:
- [ ] It would be better to have a descriptive README file which is specifically to the project instead of having the Microverse template. The template is just a guide for us to create our own unique documentation